sources/react/index-header.jsx

```diff
@@ -150,5 +150,5 @@ export default class Gridicon extends PureComponent {
 
 		switch ( icon ) {
 			default:
-				svg = <svg height={ size } width={ size } />;
+				svg = <svg height={ size } width={ size } { ...otherProps } />;
```

Comment:
Ah reading though again we'll want to omit className since we do have special handling for it already